app/components/input-new-form-information.hbs

```diff
@@ -0,0 +1,32 @@
+<AuModal
+  @title="Geef een naam en beschrijving"
+  @modalOpen={{true}}
+  @closeModal={{this.cancelFormCreation}} as |Modal| >
+  <Modal.Body>
+    <div class="au-o-grid au-o-grid--small">
+      <div class="au-o-grid__item au-1-1">
+        <AuLabel for="input-filelabel">
+          Naam
+        </AuLabel>
+        <AuInput
+          id="input-filelabel"
+          value={{this.formLabel}}
+          {{on "change" this.handleLabelChange}}
+        />
+      </div>
+      <div class="au-o-grid__item au-1-1">
+        <AuLabel for="input-filecomment">Beschrijving</AuLabel>
+        <AuTextarea
+          @width="block"
+          id="input-filecomment"
+          rows="10"
+          value={{this.formComment}}
+          {{on "input" this.handleCommentChange}}
+        />
+      </div>
+    </div>
+  </Modal.Body>
+  <Modal.Footer>
+    <AuButton {{on 'click' this.updateForm}}>Maak formulier aan</AuButton>
+  </Modal.Footer>
+</AuModal>
```

Comment:
Could we also use this component when editing the name and description of the form? Its the same modal only the actions are different but that can be solved with a callback maybe